server/handlers/queue.go

```diff
@@ -83,8 +85,25 @@ func Publish(c *gin.Context) {
 		c.JSON(http.StatusRequestEntityTooLarge, gin.H{"error": "body too large"})
 		return
 	}
+	attributes := c.GetHeader("attributes")
```

Comment:
hmmm, I think it's not good to put all attributes in one field. I prefer to splitting into multi fields like:

```
lmstfy-attribute-foo: bar
lmstfy-attribute-foz: baz
```
then we got the attributes `foo:bar` and `foz:baz`

Reply:
yes, agreed